intelmq/bots/outputs/xmpp/output.py

```diff
@@ -1,53 +1,87 @@
-# -*- coding: utf-8 -*-
-import json
+"""
+XMPP Output Bot
+Connects to a XMPP Server and sends data to a user.
+
+TODO: Introduce Multi User Chat like in XMPP Collector
+
+Requires Python >= 3.4
+Requires sleekxmpp >= 1.0.0-beta5
+
+Copyright (C) 2016 by Bundesamt für Sicherheit in der Informationstechnik
+Software engineering by Intevation GmbH
+
+Parameters:
+xmpp_user: string
+xmpp_server: string
+xmpp_password: boolean
+# TODO xmpp_room: string
+# TODO xmpp_room_password: string
+# TODO xmpp_room_nick: string
+"""
+
 import sys
 
 from intelmq.lib.bot import Bot
+
 try:
     import sleekxmpp
 except ImportError:
     sleekxmpp = None
 
 
-class XMPP(Bot):
-
+class XMPPOutputBot(Bot):
     def init(self):
         if sleekxmpp is None:
             self.logger.error('Could not import sleekxmpp. Please install it.')
             self.stop()
 
-        self.xmpp = XMPPBot(self.parameters.xmpp_user + '@' + self.parameters.xmpp_server,
-                            self.parameters.xmpp_password,
-                            self.send_message,
-                            self.logger)
+        self.xmpp = XMPPClientBot(self.parameters.xmpp_user + '@'
+                                  + self.parameters.xmpp_server,
+                                  self.parameters.xmpp_password,
+                                  self.logger)
         self.xmpp.connect(reattempt=True)
         self.xmpp.process()
 
     def process(self):
         event = self.receive_message()
 
+        receiver = self.parameters.xmpp_to_user + '@' \
+                   + self.parameters.xmpp_to_server
+
         if event is None:
             self.acknowledge_message()
+        else:
+            jevent = event.to_json()
+
+            try:
+                # TODO: proper error handling. Right now it cannot be
+                # detected if the message was sent successfully.
+                self.logger.debug("XMPP Trying to send Event: %r , to %r",
+                                  jevent,
+                                  receiver)
+                self.xmpp.send_message(mto=receiver, mbody=jevent)
+            except sleekxmpp.exceptions.XMPPError as err:
```

Comment:
From my POV it's not necessary as sleekxmpp is imported.

Are you experiencing problems with this LOC?